cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -175,9 +182,22 @@ def index_item(item, skip_index=False, groups_usage_info=None):
                 return
 
             item_content_groups = None
+
+            if item.category == SPLIT_TEST_COMPONENT_TYPE:
+                if groups_usage_info is None and item.has_children:
+                    groups_usage_info = {}
+                for vertical in item.get_children():
+                    group_id = int(vertical.display_name.split(" ")[2])
```

Comment:
Is this really the only way to get group_id? Splitting display name strings by space?